Modules/Numerics/Statistics/include/itkWeightedMeanSampleFilter.hxx

```diff
@@ -33,17 +33,6 @@ WeightedMeanSampleFilter<TSample>::WeightedMeanSampleFilter()
   this->ProcessObject::SetNthInput(1, nullptr);
 }
 
-template <typename TSample>
-void
-WeightedMeanSampleFilter<TSample>::PrintSelf(std::ostream & os, Indent indent) const
-{
-  Superclass::PrintSelf(os, indent);
-  // m_Weights
-  os << indent << "Weights: " << this->GetWeightsInput() << std::endl;
```

Comment:
Parent prints?

Reply:
Nothing: `MeanSampleFilter` has no ivar members; the latters parent is ProcessObject. I tried to look at the definition of the method and I did not find it. Despite being really awkward, have not investigated further.

Reply:
`itkSetGetDecoratedInputMacro(Weights, WeightArrayType);` creates a special input to the filter. If ProcessObject prints a pointer to it, that should be fine, and we can remove this PrintSelf.

Reply:
OK, thanks Dženan. Then I understand that we're OK. Can you resolve the thread if you think we're OK or after checking it if you wish?